aws/s3.tf

```diff
@@ -123,3 +123,58 @@ module "s3_bucket" {
 
   tags = local.tags
 }
+# aws S3 backup
+resource "aws_backup_vault" "datagrok_public_vault" {
+  name = "datagrok_public_vault"
+}
+
+resource "aws_iam_role" "backup_role" {
+  name = "s3-backup-role"
+
+  assume_role_policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Effect    = "Allow"
+        Principal = {
+          Service = "backup.amazonaws.com"
+        }
+        Action = "sts:AssumeRole"
+      }
+    ]
+  })
+}
+
+# Attach an IAM policy to the backup role that allows  performing AWS Backup jobs
+resource "aws_iam_role_policy_attachment" "backup_policy_attachment" {
+  policy_arn = "arn:aws:iam::aws:policy/service-role/AWSBackupServiceRolePolicyForBackup"
```

Comment:
Is it secure enough?

Reply:
why not? arn:aws:iam::aws:policy/service-role/AWSBackupServiceRolePolicyForBackup is all known policy

Reply:
this policy is needed for Iam role can do backups. is it something wrong?